tests/test_to_be_removed.py

```diff
@@ -0,0 +1,2 @@
+def test_to_be_removed():
```

Comment:
Does this need to be removed?

Reply:
Yes, I just included this dummy test because otherwise the CI will fail if there's not at least one existing test.
I guess when this is merged, @alessandrostockman can just rebase on the `main` branch and remove this file in https://github.com/Farama-Foundation/Jumpy/pull/5.

Reply:
@kir0ul Sorry, I just realised that jax only supports 3.7+ so that would fail on install. Could you revert it back

Reply:
Done!